sqld/src/query_response_builder.rs

```diff
@@ -0,0 +1,756 @@
+use std::io;
+use std::ops::{Deref, DerefMut};
+
+use anyhow::anyhow;
+use rusqlite::types::ValueRef;
+use serde::{Serialize, Serializer};
+use serde_json::ser::{CompactFormatter, Formatter};
+
+#[derive(Debug, thiserror::Error)]
+#[error(transparent)]
+pub struct QueryResultBuilderError(#[from] anyhow::Error);
```

Comment:
What is the motivation behind this type? Could we just use `anyhow::Error` without wrapping it?

Reply:
getting rid of anyhow soon

Reply:
Oh, I didn't know about that, what are we going to replace anyhow with?

Reply:
Nothing